2022-09-september/README.md

```diff
@@ -0,0 +1,114 @@
+- [Hack Orlando](https://hackorlando.org/) ~ October 29 in Orlando, FL
+- [poolesville_hacks 2](https://poolesvillehacks.tech/) ~ November 18-20 in Poolesville, MD
+- [Lion City Hacks](https://lioncityhacks.com/) ~ December 3 in Singapore
+- [HackOC](https://hackoc.org/) ~ Date TBC in Orange County, CA
+
+We keep an directory of upcoming high school hackathons at [hackathons.hackclub.com](https://hackathons.hackclub.com), strongly recommend it!
+
+It's been amazing seeing Hack Clubbers are working together in [#hackathon-organizers](https://hackclub.slack.com/archives/C03QSGGCJN7) to create all these magical hackathons. In that Slack channel we talk about raising money, prizes, food, swag & more! As someone organising my own hackathon, it's means a lot to have a group of friends there to help me along the way.
+
+<img src="https://ca.slack-edge.com/T0266FRGM-UBN4ED8MC-293374a1dd8b-48" width="16"> Theo, a Hack Club alum who's cold emailing made the [Elon Musk AMA](https://www.youtube.com/watch?v=riru9OzScwk) possible, has been kindly helping edit sponsor's pitches for example:
+
+<img width="420" alt="Screenshot 2022-09-24 at 10 41 46 PM" src="https://user-images.githubusercontent.com/39828164/192104131-37b2257e-4bf3-40c3-893d-9200bd732005.png">
+
+
+On a related note, just last week, we shared **[🎬 The Assemble Documentary](https://www.youtube.com/watch?v=PnK4gzO6S3Q)**: a film on Assemble to help answer the question “what is a hackathon?“. It showcases the magical experience of attending a hackathon, and our hope is that it can help organisers share what’s in store for attendees at their hackathons.
+
+<a href="https://www.youtube.com/watch?v=PnK4gzO6S3Q">
+  <img src="https://user-images.githubusercontent.com/39828164/192104361-fb7b2dd1-14d5-4297-9aa0-b37cd96dc912.jpg" width="400px" />
+</a>
+
+So that's what's been happening this month! Looking to escape school & get more involved? Here a couple of great ways to get more involved in the community at the moment:
+
+- 👋 Join a [Hack Night](https://hackclub.com/night/) call! Every Saturday evening (in America, for me its Sunday morning!), we all jump on a Zoom call to hang out and code together. 
+
+- 🌈 Organise a hackathon using [Hack Club Bank](https://hackclub.com/bank), with the help of fellow organisers in [#hackathon-organizers](https://hackclub.slack.com/archives/C03QSGGCJN7) and with one of [our $500 high-school hackathon grants](https://hackclub.com/hackathons/grant). 
+
+- 🎒 Bring together of friends together & bring the hack to your school by starting a club: https://apply.hackclub.com!
+
+- 🍃 Procrastinate by playing games in [Sprig Gallery](https://sprig.hackclub.com/gallery), or, even better, "productively procrastinate" by [making a Sprig game](https://editor.sprig.hackclub.com/).
+
+- 🔥 Start a streak on [Scrapbook](https://scrapbook.hackclub.com). (Personally, I've just started [a new streak on mine](https://scrapbook.hackclub.com/sampoder) as I learn how to use RedwoodJS!)
```

Comment:
lol sam your streak is gone D:

i'd link to javi & his 365+ (even if scrappy counted wrong, lol) streak

Reply:
![](https://emoji.slack-edge.com/T0266FRGM/jankman/09159ada2ee32987.png)
![](https://emoji.slack-edge.com/T0266FRGM/sadglas/a161abce92be7f7e.png)